modules/migration-debugging-velero-admin-webhooks.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/troubleshooting.adoc
+
+[id="migration-debugging-velero-admin-webhooks_{context}"]
+= Issues with Velero and admission webhooks
+
+Velero has limited abilities to resolve admission webhook issues, so you might need to use an additional Velero plug-in or make changes to the workload itself.
+
+For example, both Knative resources and IBM AppConnect resources might not restore properly with Velero.
```

Comment:
maybe add link references as in the bug?
https://github.com/vmware-tanzu/velero/issues/4847

Reply:
Added links for the Knative resources and IBM AppConnect resources issues.

Reply:
@RichardHoch The IBM issue was closed because there wasn't anything to fix in OADP. That's one of the issues that led to recognizing the need to document this sort of limitation.